.github/workflows/release.yml

```diff
@@ -0,0 +1,118 @@
+
+# This Workflow performs a full release of Che Code
```

Comment:
Task as described was:
" rename image-publish-insiders.yaml to image-publish.yaml and have it handle BOTH main branch -> insiders and next, and add code for handling 7.yy.x -> 7.yy.z releases"

Reply:
but I suppose in other repos we have a release.yml and a next-build.yml...

https://github.com/che-incubator/kubernetes-image-puller-operator/tree/main/.github/workflows
https://github.com/che-incubator/kubernetes-image-puller/tree/main/.github/workflows
https://github.com/che-incubator/chectl/tree/main/.github/workflows

Reply:
can you merge the functionality of  image-publish-insiders.yaml into the new release.yml so we don't have nearly identical duped files? or @azatsarynnyy are you OK with having a release.yml and a  image-publish-insiders.yaml ?

Reply:
Sounds like Artem is OK with two files, so let's go with what you have right now.

Reply:
I'm OK with both variants. If two files are more clear from your PoV - let's use that approach.